### ISSUE TITLE
Error using gpuArray/reshape

### ISSUE THREAD

**Issue by tp655998** [matlab-deep-learning/pretrained-yolo-v4#4]
when i run the section 'Evaluate Model', the error happened:

Error using gpuArray/reshape
Number of elements must not change. Use [] as one of the size inputs to automatically calculate the appropriate size for that dimension.

Error in dlarray/reshape (line 44)
objdata = reshape(objdata, varargin{:});

Error in helper.postprocess (line 27)
    currentFeatureMap = reshape(currentFeatureMap,h,w,5+numClasses,numAnchors,batchsize);

Error in detectYOLOv4 (line 39)
[bboxes,scores,labels] = helper.postprocess(outFeatureMaps, anchors, ...

Error in yolov4TransferLearn (line 347)
    [bboxes, scores, labels] = detectYOLOv4(net, image, anchors, classNames, executionEnvironment);

I training the model use six classes 
classNames = {'bicycle', 'bus', 'car', 'motorcycle', 'person', 'truck'};

**Comment by sbairagy-MW** [comment]
@tp655998 The above mentioned error happened due to one bug in estimating the number of classes from the given 'classNames' cell array. I have fixed the issue. Now, you can run the section with your data.

**Comment by JiaDQ** [comment]
@sbairagy-MW 我也遇到了相同的问题，请问这个问题是怎么解决的呢？

**Comment by JiaDQ** [comment]
@tp655998 请问这个问题是怎么解决的？
